src/crypto/tls/ech_provider.go

```diff
@@ -179,6 +179,17 @@ func (keySet *EXP_ECHKeySet) GetDecryptionContext(rawHandle []byte, version uint
 	// Compute the decryption context.
 	opener, err := key.setupOpener(handle.enc, suite)
 	if err != nil {
+		if err.Error() == "hpke: invalid KEM public key" {
```

Comment:
Mmm.. can we make the string a variable to be compared?

Reply:
Done.